packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -580,6 +599,30 @@ export class ResourceService {
       const newResourcesMap = new Map<string, Resource>();
 
       for (const newService of newServices) {
+        const adminPathWithoutLastFolder = originalResourceAdminPath?.substring(
+          0,
+          originalResourceAdminPath.lastIndexOf("/") + 1
+        );
+
+        const serverPathWithoutLastFolder =
+          originalResourceServerPath?.substring(
+            0,
+            originalResourceServerPath.lastIndexOf("/") + 1
+          );
+
+        const baseAdminPath =
+          !originalResourceId || !originalResourceAdminPath
+            ? ""
+            : !originalResourceAdminPath.includes("/")
+            ? `${newService.name}-admin`
+            : `${adminPathWithoutLastFolder}${newService.name}-admin`;
```

Comment:
Did you check we are not losing the "/" between the LastFolder and the newService.name?